frontend/app/src/people/utils/AssignedUnassignedBounties.tsx

```diff
@@ -129,6 +156,13 @@ const Bounties = (props: BountiesProps) => {
   const { ui } = useStores();
   return (
     <>
+      {props.uuid && props.name && (
+        <OrganizationWrap>
+          <Link to={`/org/tickets/${props.uuid}`} target="_blank">
+            <OrganizationText>{props.name}</OrganizationText>
```

Comment:
same as the comment above but removing props.name -> name and props.uuid -> uuid would make code cleaner